docs/sources/articles/host_integration.md

```diff
@@ -65,11 +65,14 @@ a new service that will be started after the docker daemon service has started.
     [Unit]
     Description=Redis container
     Author=Me
+    Requires=docker.service
     After=docker.service
 
     [Service]
     Restart=always
     ExecStart=/usr/bin/docker start -a redis_server
+    # for more options, use 'run' instead of 'start', but not suggested
```

Comment:
Oops, you're right, Sven. I missed that was a code block. 

I'm +1 on @SvenDowideit 's rephrase. @dlintw Would you like us to carry this PR so we can fix the language for you?